### ISSUE TITLE
image is missing git

### ISSUE THREAD

**Issue by 1ma** [rawkode/docker-php#1]
composer needs git in order to fetch packages when the user requests a non stable version or uses the `--prefer-source` option

```
root@c8a44d93738c:~# composer require --prefer-dist slim/slim
Do not run Composer as root/super user! See https://getcomposer.org/root for details
Using version ^3.8 for slim/slim
./composer.json has been created
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 6 installs, 0 updates, 0 removals
    Failed to download psr/container from dist: The zip extension and unzip command are both missing, skipping.
A php.ini file does not exist. You will have to create one.
    Now trying to download from source
  - Installing psr/container (1.0.0): Cloning b7ce3b1764

Installation failed, deleting ./composer.json.

                                                                                                                                 
  [RuntimeException]                                                                                                             
  Failed to clone https://github.com/php-fig/container.git, git was not found, check that it is installed and in your PATH env.  
  sh: 1: git: not found                                                                                                          
                                                                                                                                 

require [--dev] [--prefer-source] [--prefer-dist] [--no-progress] [--no-suggest] [--no-update] [--no-scripts] [--update-no-dev] [--update-with-dependencies] [--ignore-platform-reqs] [--prefer-stable] [--prefer-lowest] [--sort-packages] [-o|--optimize-autoloader] [-a|--classmap-authoritative] [--apcu-autoloader] [--] [<packages>]...
```

**Comment by rawkode** [comment]
Good catch, @1ma ! Feel free to submit a PR if you have the time, if not - I'll have this done this afternoon.

Thanks for reporting :+1:

**Comment by rawkode** [comment]
Closed by #3
